app/reducers/toasts.ts

```diff
@@ -12,24 +13,25 @@ const initialState = {
 type State = {
   items: Array<Toast>;
 };
-const toasts = produce<State>((newState: State, action: any): void => {
-  switch (action.type) {
-    case Toasts.TOAST_ADDED:
-      newState.items.push(action.payload);
-      break;
+const toasts: Reducer<State> = (state = initialState, action) =>
+  produce(state, (newState) => {
+    switch (action.type) {
+      case Toasts.TOAST_ADDED:
+        newState.items.push(action.payload);
+        break;
 
-    case Toasts.TOAST_REMOVED: {
-      const toast = newState.items.find((t) => t.id === action.payload.id);
+      case Toasts.TOAST_REMOVED: {
+        const toast = newState.items.find((t) => t.id === action.payload.id);
```

Comment:
Accidental or intentional?

Reply:
I changed the function creation slightly to make it work with typescript, that's why the indentation changed.